mitodl/mitxpro-theme
# style: add trademark logos

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/5719

### Description (What does it do?)
- Adds the trademark logos

### Screenshots (if appropriate):
**Learning MFE (Desktop)**
<img width="1791" alt="image" src="https://github.com/user-attachments/assets/9e02a465-bc28-4e2a-b7bf-a06892338ed7">

Learning MFE (Mobile)
<img width="430" alt="image" src="https://github.com/user-attachments/assets/7da4bf56-8cff-456d-895d-1c47afaecd91">

**Dashboard (Desktop)**
<img width="1792" alt="image" src="https://github.com/user-attachments/assets/4803eefc-ae61-4c4d-b892-f71699bfab37">

**Dashboard (Mobile)**
<img width="428" alt="image" src="https://github.com/user-attachments/assets/f680d81c-ccae-4a7c-8d65-8497bff8868b">


**Studio**
<img width="1792" alt="image" src="https://github.com/user-attachments/assets/b3406a6e-85f7-41d1-8282-27753a8bee62">

### How can this be tested?
- Setup this theme
- Check that all the logos are updated - You might need to setup some config variables to proper URLs in `private.py` and `.env.development` for MFEs to make these properly work. We have already set these in our deployments so there would be no problem there
